web/src/admin/AdminInterface/AdminSidebar.ts

```diff
@@ -0,0 +1,176 @@
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import { VERSION } from "@goauthentik/common/constants";
+import { me } from "@goauthentik/common/users";
+import { authentikConfigContext } from "@goauthentik/elements/AuthentikContexts";
+import { AKElement } from "@goauthentik/elements/Base";
+import { ID_REGEX, SLUG_REGEX, UUID_REGEX } from "@goauthentik/elements/router/Route";
+import { spread } from "@open-wc/lit-helpers";
+
+import { consume } from "@lit-labs/context";
+import { msg, str } from "@lit/localize";
+import { TemplateResult, html, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import { map } from "lit/directives/map.js";
+
+import {
+    AdminApi,
+    CapabilitiesEnum,
+    type Config,
+    CoreApi,
+    type SessionUser,
+    UiThemeEnum,
+    type UserSelf,
+    Version,
+} from "@goauthentik/api";
+
+@customElement("ak-admin-sidebar")
+export class AkAdminSidebar extends AKElement {
+    @property({ type: Boolean })
+    open = true;
+
```

Comment:
These are the only bits we care about in either of the calls to `AdminApi()` and `me()`.